easybuild/easyblocks/generic/tarball.py

```diff
@@ -55,15 +55,17 @@ def build_step(self):
         """
         pass
 
-    def install_step(self):
+    def install_step(self, src=None):
+        """Install by copying from specified source directory (or 'start_dir' if not specified)."""
+        if src is None:
+            src = self.cfg['start_dir']
 
-        src = self.cfg['start_dir']
         # shutil.copytree cannot handle destination dirs that exist already.
         # On the other hand, Python2.4 cannot create entire paths during copytree.
         # Therefore, only the final directory is deleted.
         rmtree2(self.installdir)
         try:
             # self.cfg['keepsymlinks'] is False by default except when explicitly put to True in .eb file
-            shutil.copytree(src,self.installdir, symlinks=self.cfg['keepsymlinks'])
-        except:
-            self.log.exception("Copying %s to installation dir %s failed" % (src,self.installdir))
+            shutil.copytree(src, self.installdir, symlinks=self.cfg['keepsymlinks'])
+        except OSError, err:
+            self.log.exception("Copying %s to installation dir %s failed: %s" % (src, self.installdir, err))
```

Comment:
ah, this should be `log.error`, for now

after the 2.0 release, we will deprecate `log.error` and switch to `raise EasyBuildError`